tests/fermi/test_fermionic.py

```diff
@@ -282,6 +283,11 @@ def test_add_fermi_words_and_sentences(self, w, s, res):
         (fw3, (1 + 3j), FermiSentence({fw3: 1, fw4: (1 + 3j)})),  # complex
         (fw1, np.array([5]), FermiSentence({fw1: 1, fw4: 5})),  # numpy array
         (fw2, pnp.array([2.8]), FermiSentence({fw2: 1, fw4: 2.8})),  # pennylane numpy array
+        (
+            fw1,
+            pnp.array([2, 2])[0],
```

Comment:
Do we need tests for `np`?

Reply:
We have the test for a numpy array above (line 284 in this case). 

Indexing a `numpy` array just gives you back the element of the array, not a 0d tensor like `pennylane.numpy`, so that is covered by the `float`, `complex` and `int` tests. So I don't see a need for `np.array([0, 1])[0]`.